pkg/alertmanager/operator.go

```diff
@@ -1397,6 +1402,28 @@ func checkWechatConfigs(
 	return nil
 }
 
+func checkWebexConfigs(
+	ctx context.Context,
+	configs []monitoringv1alpha1.WebexConfig,
+	namespace string,
+	key string,
+	store *assets.Store,
+	amVersion semver.Version,
+) error {
+	for i, config := range configs {
```

Comment:
we need a check on the amVersion too (e.g. return an error if version < v0.25.0 similar to the check in `checkTelegramConfigs()`).